docs/harmony.mess

```diff
@@ -10,9 +10,23 @@ First, you will want to inherit from ``org.shirakumo.fraf.trial.harmony:settings
 
 See ``harmony:make-simple-server`` for more information on accepted initargs.
 
-Second, you will want to pre-load sound files. The ``trial-harmony`` system provides assets and resources to encapsulate the Harmony objects you need to manage. To import sounds and simple music tracks, use the ``org.shirakumo.fraf.trial.harmony:sound`` to define your assets. If you want to use Harmony's environments system to do horizontal mixing, define the environment using the ``org.shirakumo.fraf.trial.harmony:environment`` asset.
+Second, you will want to pre-load sound files. The ``trial-harmony`` system provides assets and resources to encapsulate the Harmony objects you need to manage. To import sounds and simple music tracks, use the ``org.shirakumo.fraf.trial.harmony:sound`` to define your assets:
 
-In order to play back a sound, just access its resource and play it back:
+:: common lisp
+(define-asset (my-pool my-sound) trial-harmony:sound
```

Comment:
This is outdated, you should use trial's `audio-file` or `sound-bank` (for multiple effects that should be randomly picked between).